src/test/coins_tests.cpp

```diff
@@ -773,8 +773,10 @@ static void CheckAddCoinBase(CAmount base_value, CAmount cache_value, CAmount mo
 template <typename... Args>
 static void CheckAddCoin(Args&&... args)
 {
-    for (const CAmount base_value : {ABSENT, SPENT, VALUE1})
+    for (const CAmount base_value : {ABSENT, SPENT, VALUE1}) {
```

Comment:
This is masking a legitimate bug, it just happens to be ok for now because of the types involved.

It shouldn't be marked nolint, rather it should be fixed: drop the forward and pass by `const Args&`.

Reply:
I've removed the commits from #31364